js/web/lib/wasm/jsep/webgpu/ops/conv.ts

```diff
@@ -136,12 +136,32 @@ const conv2d = (context: ComputeContext, inputs: readonly TensorView[], attribut
   // check attributes
 
   // const hasPreluActivationWeights = false; /* TODO: add support for prelu activation weights */
+  const isChannelsLast = attributes.format === 'NHWC';
   if (attributes.group !== 1) {
-    context.compute(createGroupedConvProgramInfo(inputs, adjustedAttributes));
+    if (isChannelsLast && inputs[1].dims[0] === attributes.group && inputs[1].dims[1] === 1 &&
```

Comment:
Do we need to check unit strides here?

Reply:
`createGroupedConvVectorizeProgramInfo` supports any strides so it won't have error.

Reply:
Thanks for confirming that.